public-api/config/config_staging.json

```diff
@@ -8,6 +8,8 @@
     "workbenchApiBaseUrl": "https:\/\/api-dot-all-of-us-rw-staging.appspot.com",
     "publicApiKeyForErrorReports": "AIzaSyAkMIMIzUwv02RBK-A7cE1PbPpDJ2MTNtk",
     "debugEndpoints": false,
-    "projectId": "aou-db-staging"
+    "projectId": "aou-db-staging",
+    "gaId": "UA-116298798-4",
+    "clientId": "657299777109-kvb5qafr70bl01i6bnpgsiq5nt6v1o8u.apps.googleusercontent.com"
```

Comment:
It is a required in `trackEventToGoogleAnalytics` method - it's one of the required elements of the post data object that goes to Google.

Reply:
I think this is conflating an oauth client ID (used for login) and a Google Analytics client ID, used to identify distinct users. I don't think we want to be putting our oauth client ID here.

https://support.google.com/analytics/answer/6205850?hl=en#clientid-userid

Here's the comment from the relevant section of the Python GA sample code:

```
        # Anonymous Client Identifier. Ideally, this should be a UUID that
        # is associated with particular user, device, or browser instance.
        'cid': '555',
```

I think ideally this would match whatever value is used client-side for the user's GA session, but I'm not sure whether we can access that.

Reply:
Alternatively, just hardcode it to a value as in the sample docs, e.g. `555` with a TODO to somehow leverage the user's browser-based client ID.

Reply:
Ah ... I did conflate these ids. Will look into this a little more. Ideally, we'd like to know the user behind the API call so GA can appropriately group these events with other user-specific events.